components/cf-app-push/frontend/src/view/deploy-app-workflow/deploy-step-deploying/deploy-step-deploying.service.js

```diff
@@ -234,6 +236,7 @@
 
         sourceUserInput.fileTransfers = metadata.files;
         metadata.files = metadata.files.length;
+        metadata.type = metadata.files === 1 ? 'archive' : 'filefolder';
```

Comment:
The backend checks the file extension to determine if it is an archive file - rather than basing this on there being only 1 file.